src/wizard/wizard.scss

```diff
@@ -56,107 +55,112 @@
         text-align: center;
         z-index: 1;
         user-select: none;
+        @include themed {
+          border: 1px solid t(iui-color-background-5);
+          background-color: t(iui-color-background-1);
+          color: t(iui-text-color-muted);
+        }
+      }
+
+      &.iui-current {
```

Comment:
Why these CSS rules are needed now?

Reply:
Oh I see it is moved from bottom. Can it be on the bottom because now looks like a big change and we would need to review every line?

Reply:
Ok, I think I put it back where it was originally.  Only change I made is from `> .iui-current` to `.iui-step.iui-current`.

Reply:
Any chance we can change this PR to only tackle the responsiveness (#77)? Because right now, it's very hard to review

Lets save the big changes for #109